index.js

```diff
@@ -91,19 +94,42 @@ if (process.stdout.isTTY) {
     debug('Branch to Diff: ', diffBranch);
   }
 
-  let committedGitFiles = [];
-
+  let isdiffBranchExisted = false;
   try {
-    committedGitFiles = fetchGitDiff(diffBranch);
-    debug('Committed GIT files: ', committedGitFiles);
-  }
-  catch(err) {
+    isdiffBranchExisted = checkForBranchExistence(baseBranch,remote);
+    debug('Check whether branch is existed: ', diffBranch);
```

Comment:
This PR is good to go with a success log stating that the base branch is existing.

Reply:
It would make the debugging process a bit easier.

Reply:
How about now? 😊

Reply:
Cool 👍